manifest.json

```diff
@@ -4218,6 +4218,28 @@
                 }
             }
         },
+        "Banane9.ComponentAttacherRecents": {
+            "name": "ComponentAttacherRecents",
+            "description": "Adds a special category of your recently used components to the Component Attacher.",
+            "category": "Developers",
+            "sourceLocation": "https://github.com/Banane9/NeosComponentAttacherRecents/",
+            "authors": {
+                "Banane9": {
+                    "url": "https://github.com/Banane9"
+                }
+            },
+            "versions": {
+                "1.0.0": {
```

Comment:
Non-blocking notes:
AssemblyCopyright year still set to 2022
AssemblyDescription appears to be still set for 'component attacher favorites'